components/exercises/chapter-5-oscar/Everything/some.test.tsx

```diff
@@ -0,0 +1,15 @@
+import { some } from './some';
+
+describe('some', () => {
+  it('should return true if each satisfies the condition', () => {
+    const arr = [2, 4, 6, 8];
+    const callBack = (num: number) => num < 10;
+    expect(some(arr, callBack)).toBe(true);
+  });
+
+  it('should return false if at least one element satisfies the condition', () => {
```

Comment:
it('should return false if none of the element satisfies the condition', () => {

Reply:
refactored